### ISSUE TITLE
Broken when using Pyinstaller

### ISSUE THREAD

**Issue by OuFinx** [workhorsy/py-cpuinfo#80]
Hello. I need show brand of my processor. 
```
import cpuinfo

infocpu = cpuinfo.get_cpu_info()
ProcessorName = infocpu['brand']
```
I check it, and everything work. After, i compile exe file using Pyinstaller. 
Then, i run my exe file (main.exe), program working but dont closed, i go to Task Manager and see pictures where my programs clone process of program. 
And after 1 minutes my CPU have load 100%. 

Why program don't work correctly?

# In your bug report please include:
* CPU architecture Intel64
* Operating System Windows 10
* Python version 3.5.2
* Version of py-cpuinfo 3.3.0

**Comment by workhorsy** [comment]
I'll have to try it with Pyinstaller and see if I can figure out the problem. Thanks.

**Comment by workhorsy** [comment]
It looks like it is failing when trying to run a function in a separate process. Obviously doing this won't work when running in a compiled exe that does not have access to the installed python process:

```python
returncode, output = run_and_get_stdout([sys.executable, "-c", "import cpuinfo; print(cpuinfo.actual_get_cpu_info_from_cpuid())"])
if returncode != 0:
	return {}
info = b64_to_obj(output)
```

**Comment by OuFinx** [comment]
This problem can be somehow bypassed?

**Comment by workhorsy** [comment]
It looks like it is a documented problem with Pyinstaller on Windows:
https://github.com/pyinstaller/pyinstaller/wiki/Recipe-Multiprocessing

I think I've almost got it working. I just need to test it on a few more machines, and clean the code up:
https://github.com/workhorsy/py-cpuinfo/commits/pyinstaller3

**Comment by workhorsy** [comment]
Basically there are limitations in the way Pyinstaller works with multiprocessing. So you even with these fixes, you would also have to change the example to be inside a main guard, and freeze support:

```python

# main.py
# Put everything in a main guard or it may execute twice when the process forks 
if __name__ == '__main__':
	import multiprocessing
	import get_cpu_info

	# Call freeze_support or Pyinstaller will recursively fork infinite processes forever
	multiprocessing.freeze_support()
	print(get_cpu_info.get_cpu_info())
```

Then it should work as expected:
```bash
pyinstaller main.py --onefile
```

Once I get this branch done anyway.

**Comment by OuFinx** [comment]
What is this "import get_cpu_info" ?

Its not "import cpuinfo" ?

How i can install it?

Thx

**Comment by workhorsy** [comment]
Oh sorry. I made a typo

**Comment by OuFinx** [comment]
It's don't work for me.
Process with program still clone every 1 seconds in task manager

**Comment by workhorsy** [comment]
Are you testing the https://github.com/workhorsy/py-cpuinfo/commits/pyinstaller3 branch?

**Comment by OuFinx** [comment]
Now everything work.
Thanks so much!

**Comment by workhorsy** [comment]
Nice. I'll have to do a few more tweaks, and more tests before merging into main. Thanks.

**Comment by workhorsy** [comment]
Fixed is PR #83
